api/src/routers/meals.js

```diff
@@ -61,7 +135,7 @@ mealsRouter.put("/:id", async (req, res, next) => {
 mealsRouter.delete("/:id", async (req, res, next) => {
   try {
     const id = req.params.id;
-    const deletedMeal = await knex("Meal").where("id", id).del();
+    const deletedMeal = await knex("meal").where("id", id).del();
```

Comment:
Apply the suggestions in all applicable places

```suggestion
    const deletedMeal = await knex("meal").where({id}).del();
```